web/beacon-app/src/features/projects/components/RenameProject/RenameProjectForm.tsx

```diff
@@ -0,0 +1,34 @@
+import { t } from '@lingui/macro';
+import { Button, TextField } from '@rotational/beacon-core';
+import { ErrorMessage, Form, FormikHelpers, FormikProvider } from 'formik';
+
+import { Project } from '../../types/Project';
+import { useUpdateProjectForm } from '../../types/updateProjectFormService';
+import type { UpdateProjectDTO } from '../../types/updateProjectService';
+
+type RenameProjectModalFormProps = {
+  handleSubmit: (values: UpdateProjectDTO, helpers: FormikHelpers<UpdateProjectDTO>) => void;
+  project: Project;
+};
+
+function RenameProjectForm({ handleSubmit, project }: RenameProjectModalFormProps) {
+  const formik = useUpdateProjectForm(handleSubmit, project);
```

Comment:
This fix the rename project validation issue

Reply:
Should we add a story to make the `Save` button disabled until a new project name is entered?

<img width="1572" alt="Screenshot 2023-05-15 at 10 14 24 AM" src="https://github.com/rotationalio/ensign/assets/94616884/a7481ad9-58c9-4c93-a347-d52649754596">

Reply:
I'm not sure it is necessary to add it since when the user is trying to click on the `save button` he will get an error message.